vofs/vofs/CadcCache.py

```diff
@@ -930,8 +930,8 @@ def flush(self):
 
             # Look for write failures.
             if self.flushException is not None:
-                raise self.flushException[0], self.flushException[1], \
-                    self.flushException[2]
+                raise self.flushException[0]('{} - {}'.format(self.flushException[1],
```

Comment:
flushException is set by calling sys.exc_info() that returns that tuple - hence we expect that.